decidim-system/app/views/decidim/system/api_users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% add_decidim_page_title(t(".manage")) %>
+
+<% provide :title do %>
+  <h1 class="h1"><%= t ".manage" %></h1>
+<% end %>
+
+<p><%= simple_format(t ".explanation") %></p>
```

Comment:
Let's get rid of the wrapping `<p>` and the `simple_format` call. Te HTML formatting can be added to the string itself.